### PR TITLE
Feat/data 11 adding kafka connect chart

### PR DESCRIPTION
**Pull request by casanova-thiago**

### CODE REVIEW

File: charts/kafka-operator-instance/templates/connect.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  
+  # template:
+  #   serviceAccount:
+  #     metadata:
+  #       annotations:
+  #         eks.amazonaws.com/role-arn: arn:aws:iam::123456789010:role/data-kafka-connect
+  #   buildServiceAccount:
+  #     metadata:
+  #       annotations:
+  #         eks.amazonaws.com/role-arn: arn:aws:iam::123456789010:role/data-kafka-connect-build
```
Comment (jfmrm) [inline charts/kafka-operator-instance/templates/connect.yaml:38]: all these should come from values.yaml

Comment (jfmrm) [reply charts/kafka-operator-instance/templates/connect.yaml:38]: and it's missing the last empty line

**Review by jfmrm — CHANGES_REQUESTED** [review]

File: charts/kafka-operator-instance/templates/connect.yaml (hunk 1 above)
Comment (casanova-thiago) [reply charts/kafka-operator-instance/templates/connect.yaml:38]: So... something like this or remove this part you highlighted?

```yaml
apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaConnect
metadata:
  name: {{ .Values.connect.name }}
  namespace: {{ .Values.namespace }}
spec:
  version: {{ .Values.connect.version }}
  replicas: {{ .Values.connect.replicas }}
  bootstrapServers: {{ .Values.connect.bootstrapServers }}
  config:
    group.id: {{ .Values.connect.groupId }}
    offset.storage.topic: {{ .Values.connect.offsetStorageTopic }}
    config.storage.topic: {{ .Values.connect.configStorageTopic }}
    status.storage.topic: {{ .Values.connect.statusStorageTopic }}
  image: {{ .Values.connect.image }}
  build:
    output:
      type: docker
      image: {{ .Values.connect.image }}
    plugins:
      - name: {{ .Values.connect.plugin.name }}
        artifacts:
          - url: {{ .Values.connect.plugin.url }}
```

Comment (jfmrm) [reply charts/kafka-operator-instance/templates/connect.yaml:38]: I would say something more similar to the kafka kind to give us more freedom

Comment (casanova-thiago) [reply charts/kafka-operator-instance/templates/connect.yaml]: done! let me know if it's better now!

File: charts/kafka-operator-instance/templates/connect.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: {{ .Values.name }}
+spec:
+    {{- toYaml .Values.spec | nindent 2 }}
```
Comment (jfmrm) [inline charts/kafka-operator-instance/templates/connect.yaml:6]: now we need to change this for both Kafka and Kafka connect:
```suggestion
    {{- toYaml .Values.kafkaConnect.spec | nindent 2 }}
```

**Review by jfmrm — CHANGES_REQUESTED** [review]

File: charts/kafka-operator-instance/templates/kafka.yaml [hunk 3]
```diff
@@ -3,4 +3,4 @@ kind: Kafka
 metadata:
   name: {{ .Values.name }}
 spec:
-    {{- toYaml .Values.spec | nindent 2 }}
+    {{- toYaml .Values.Kafka.spec | nindent 2 }}
```
Comment (jfmrm) [inline charts/kafka-operator-instance/templates/kafka.yaml:6]:
```suggestion
    {{- toYaml .Values.kafka.spec | nindent 2 }}
```

File: charts/kafka-operator-instance/templates/connect.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: {{ .Values.name }}
+spec:
+    {{- toYaml .Values.KafkaConnect.spec | nindent 2 }}
```
Comment (jfmrm) [inline charts/kafka-operator-instance/templates/connect.yaml:6]:
```suggestion
    {{- toYaml .Values.kafkaConnect.spec | nindent 2 }}
```

**Review by jfmrm — APPROVED** [review]

**Pull request merged**